tools/config.go

```diff
@@ -38,7 +38,7 @@ func initConfig() {
 	viper.AddConfigPath(".")
 	err := viper.ReadInConfig()
 	if err != nil {
-		if err == err.(viper.ConfigFileNotFoundError) {
+		if errors.Is(err, viper.ConfigFileNotFoundError{}) {
```

Comment:
Is it necessary to instantiate `viper.ConfigFileNotFoundError` here? E.g.: `viper.ConfigFileNotFoundError` vs `viper.ConfigFileNotFoundError{}`

Reply:
Unfortunately yes, viper doesn't expose the error as a var here, only as a type for some reason